README.md

```diff
@@ -329,6 +330,23 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     Women Hack for Non-Profits (WHFNP) is a volunteer community that enhances the skills of women in technology, supports their development and provides opportunities to explore open source. The group matches skilled women to open source projects that are being developed for non-profits, charities and individuals to create a positive social impact.
 
+### Chatrooms
+([Go to top](#awesome4girls--))
+  
+  Online chatrooms such as slack teams, gitter, IRC etc.
+  
+  - [Women in Tech](http://witchat.github.io/)
+
+    A safe, confidential space for women who work in technology to chat and support each other. 
+
+  - [Women TechMakers](https://www.womentechmakers.com/)
+
+    Community of women in early to mid level career who want to propel their career to senior level. This slack group is pretty active, apply to join.
```

Comment:
More shortened with lesser redundancy, `Early to mid-level career women who want to propel their career to senior level. Apply to join.`